### PR TITLE
Using the RAND function in the UPDATE query

### PR DESCRIPTION
**Pull request by saitorhan**

### CODE REVIEW

File: docs/t-sql/functions/rand-transact-sql.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+meta---
```
Comment (Court72) [inline docs/t-sql/functions/rand-transact-sql.md]:
```suggestion
---
```

**Comment by Court72** [comment]
@mikeraymsft

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/functions/rand-transact-sql.md [hunk 2]
```diff
@@ -51,7 +51,12 @@ RAND ( [ seed ] )
   
 ```sql  
 SELECT RAND(100), RAND(), RAND()   
-```  
+```
+
+  When using RAND() function in an UPDATE or INSERT query, all affected rows get the same value. For example, when the following query is run, all values in the RANDEXAMPLES table will be updated to the same result.
```
Comment (rwestMSFT) [inline docs/t-sql/functions/rand-transact-sql.md]:
```suggestion
When using the RAND() function in an UPDATE or INSERT query, all affected rows get the same value. For example, when the following query is run, all values for the `RandValue` column in the `RandExamples` table are updated to the same value.

```

File: docs/t-sql/functions/rand-transact-sql.md [hunk 3]
```diff
@@ -51,7 +51,12 @@ RAND ( [ seed ] )
...
+  ```sql
```
Comment (rwestMSFT) [inline docs/t-sql/functions/rand-transact-sql.md]:
```suggestion
```sql
```

File: docs/t-sql/functions/rand-transact-sql.md [hunk 4]
```diff
@@ -51,7 +51,12 @@ RAND ( [ seed ] )
...
+UPDATE RANDEXAMPLES SET RANDVALUE = RAND()
```
Comment (rwestMSFT) [inline docs/t-sql/functions/rand-transact-sql.md]:
```suggestion
UPDATE RandExamples SET RandValue = RAND();
```

**Comment by saitorhan** [comment]
Hi,
Should I do anything else about this request or should I wait for approving?

**Comment by saitorhan** [comment]
Hi,
Should I do any thing else about this request or should I wait for approving?

**Comment by rwestMSFT** [comment]
@saitorhan We will deal with it according to our internal process. Thanks.

**Comment by rwestMSFT** [comment]
@saitorhan Because the existing article needed a refresh, we are including your change in an internal edit, and this PR will be closed when that PR is merged. You can expect to see an updated article in the next business day.

**Comment by saitorhan** [comment]
Thank you so much for this great notification.

**Pull request closed without merging**